### ISSUE TITLE
[FEATURE] Play audio from local storage

### ISSUE THREAD

**Issue by jkolczasty** [TheTimeWalker/wallpanel-android#65]
Playing audio from local storage feature would be very useful for playing notifications without delay cased of downloading mp3 file every time it's needed. This could give opportunity to build chain-commands like play notification audio + text-to-search after small delay.

E.g. instead of using command {"audio": "http://domain...."} would be perfect if one could use e.g. {"audio": "file:///storage/emulated/0/Download/somefile.mp3"}

**Comment by TheTimeWalker** [comment]
I don't think this would be feasible to do, especially with the way how the storage system on Android works including the changes from 8.0 and again from Android 10

**Comment by jkolczasty** [comment]
Ok. Then my new proposal below.

Adding 2 new commands:
audioDownload - {"audioDownload": "http://..../somefile.mp3"}
This command would download and store audio in application local storage on Android device for later use in command 

audioLocal - {"audioLocal": "somefile.mp3"}
This command would play previously downloaded audio file

How about that? :-)

**Comment by TheTimeWalker** [comment]
I'll reopen the issue just to see if that's even manageable. Sounds to me more like you'd more or less have to implement a full on audio player in the app but I'm not sure.
